Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: PORT=3000 bundle exec puma -C config/puma.rb
+sidekiq: bundle exec sidekiq -q default -q push -q pull -q mailers
```

Comment:
This'll need a docs update with recommendation to run `foreman start` instead of just `rails s`, though you may not always need sidekiq during dev so I think mostly it's fine.